modules/updates-for-hosted-control-planes.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * updates/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: CONCEPT
+[id="updates-for-hosted-control-planes_{context}"]
+= Updates for hosted control planes
+
+Updates for hosted control planes involve updating the hosted cluster and updating the node pools. For a cluster to remain fully operational during an update process, you must meet the requirements of the link:https://kubernetes.io/releases/version-skew-policy/[Kubernetes version skew policy] while you complete control plane and node updates.
+
+[id="updates-for-hosted-control-planes-hostedcluster_{context}"]
+== Updates for the hosted cluster
+
+The `spec.release` value dictates the version of the control plane. The `HostedCluster` object propagates the intended `spec.release` value to the `HostedControlPlane.spec.release` value and runs the appropriate control plane operator version.
```

Comment:
Hmm I think changing `propogates` -> `transmits` will still have the same meaning in the sentence